bugout-dev/moonstream
# "--confirmations" argument for Ethereum crawler

## Changes

Adds a `--confirmations` argument to the Ethereum crawler.




## How to test these changes?

Run a sync against a working node. Check that the sync is behind the latest block by `--confirmations`. I have tested this locally against a live geth node (mainnet).




## Related issues

https://github.com/bugout-dev/moonstream/issues/84 (partially resolved by this PR)
